modules/test/playwright/tests/configuration-admin-web/instanceSettings.spec.ts

```diff
@@ -88,8 +99,77 @@ test('LPD-35562 Enter reserved screen name', async ({page}) => {
 
 	await page.getByLabel('Last Name').fill(lastName);
 
-	await instanceSettingsPage.saveAndWaitForAlert(
-		'Error:The screen name you requested is reserved.',
-		{autoClose: false, type: 'danger'}
-	);
+	await instanceSettingsPage.saveAndWaitForAlert({
+		autoClose: false,
+		text: 'Error:The screen name you requested is reserved.',
+		type: 'danger',
+	});
+});
+
+test('LPD-38043 Assert that a site configuration overrides its instance version', async ({
+	accessibilityMenuPage,
+	instanceSettingsPage,
+	page,
+	site,
+	siteSettingsPage,
+}) => {
+	await test.step('Make sure the instance accessibility configuration is disabled', async () => {
+		await instanceSettingsPage.goToInstanceSetting(
+			'Accessibility',
+			'Accessibility Menu'
+		);
+
+		if (
+			await accessibilityMenuPage.enableAccessibilityMenuCheckbox.isChecked()
+		) {
+			await accessibilityMenuPage.enableAccessibilityMenuCheckbox.uncheck();
+
+			await instanceSettingsPage.saveAndWaitForAlert({});
+		}
+	});
+
+	await test.step('Make sure the accessibility menu is not accessible in the site scope', async () => {
+		await siteSettingsPage.goToSiteSetting(
+			'Accessibility',
+			'Accessibility Menu',
+			site.friendlyUrlPath
+		);
+
+		await page.waitForLoadState();
+
+		await expect(
+			accessibilityMenuPage.openAccessibilityMenuButton
+		).not.toBeAttached();
```

Comment:
Did you check that this assertion fails when the accessibility menu is enabled? Since this button is hidden until you tab to it, I wanted to make sure `toBeAttached` can handle it.

Reply:
Yes. That method is fine. If the configuration is enabled, the menu is always present, hidden, but present in the document.